elements/rh-band/src/rh-band.scss

```diff
@@ -5,12 +5,15 @@ $SLOT: "";
 $aside-md: 250px;
 $aside-lg: 300px;
 
+
+
 :host {
-  // Spacing values
-  --rh-band--paddingTop:                      calc(#{rh-var(container-spacer)} * 4);
-  --rh-band--paddingRight:                    #{rh-var(container-spacer)};
-  --rh-band--paddingBottom:                   calc(#{rh-var(container-spacer)} * 4);
-  --rh-band--paddingLeft:                     #{rh-var(container-spacer)};
+  // Default spacing values
+  --rh-band--padding-vertical:                 calc( #{rh-var(container-spacer)} * 4);
+  --rh-band--padding-horizontal:               calc( #{rh-var(container-spacer)} * 1);
+  // Bring them together for ease of use below
+  --rh-band--padding-full:                         #{rh-local(padding-vertical)}  #{rh-local(padding-horizontal)};
```

Comment:
I think for clarity, this should be: `--rh-band--padding` since padding is the name of property this is being assigned to.

Reply:
+1